web/templates/page/index.html.eex

```diff
@@ -1,46 +1,68 @@
-<div class="jumbotron">
-  <h2><%= gettext "Welcome to %{name}", name: "PostBot" %></h2>
-  <p class="lead">A slackbot to help you keep track of article performance</p>
-  <p>
-    <%= link to: auth_path(@conn, :request, :slack) do %>
-      <img alt="Sign in with Slack" height="40" width="172" src="https://platform.slack-edge.com/img/sign_in_with_slack.png" srcset="https://platform.slack-edge.com/img/sign_in_with_slack.png 1x, https://platform.slack-edge.com/img/sign_in_with_slack@2x.png 2x" />
-    <% end %>
-  </p>
-  <%= if @current_team do %>
-    <h2>Welcome, <%= @current_team.name %>!</h2>
-  <% else %>
-    <h2>Unauthenticated</h2>
+<div class="hero">
+  <img class="hero__logo" src="/images/post_logo.png" alt="Post"/>
```

Comment:
More descriptive alt text. Or if it's a decorative image, then it can sit as a background image.